Platform/AlderlakeBoardPkg/Library/FspsUpdUpdateLib/FspsUpdUpdateLib.c

```diff
@@ -594,7 +594,9 @@ UpdateFspConfig (
 
   if (PcdGetBool (PcdFramebufferInitEnabled)) {
```

Comment:
Are we expected to initialize Gfx during S3 wake up?

Can you add the S3 check to PcdFramebufferInitEnabled ?

if (PcdGetBool (PcdFramebufferInitEnabled) && (GetBootMode() != BOOT_ON_S3_RESUME) ) {

Reply:
updated accordingly